CMakeLists.txt

```diff
@@ -1,9 +1,8 @@
-cmake_minimum_required(VERSION 3.11)
-project(Calculator VERSION 0.1.0)
+cmake_minimum_required(VERSION 3.21)
```

Comment:
There is probably no reason why you should update the minimum version...

Unless there is some feature of CMake you are using that is only available in version 3.21... in that case then changing the minimum is fine